### DISCUSSION TITLE
Add functionality to lock header and footer items

### DISCUSSION THREAD

**Discussion by ShadyMCShadesen** [iantrich/restriction-card#59] [Ideas]
Hi,
it would be very nice if header and footer entities from, for example, "entities" and "custom: hui-entities-card" cards could also be locked.

In addition, it would be even more convenient if you could set, for example, that the button that you pressed after confirming executes the action without having to press it again.

thy^^

**Comment by iantrich** [comment]
#1 surprised it doesn't already work actually. What have you tried?

#2 not feasible

**Reply by ShadyMCShadesen** [reply]
I can lock footer and header items if i lock the whole card, but i haven't figured it out how to lock induvidual items in the footer or header segment like it is possible with the "custom:hui-entities-card" 'main' entities.

**Reply by iantrich** [reply]
Looks like we'd have to add a new option and call the `createHeaderFooterElement` function from card helpers
